src/main/java/org/reaktivity/nukleus/tls/internal/stream/TlsServerFactory.java

```diff
@@ -241,18 +235,69 @@ private MessageConsumer newNetworkStream(
         final long routeId = begin.routeId();
         final long authorization = begin.authorization();
 
-        final MessagePredicate filter = (t, b, o, l) -> true;
-        final RouteFW route = router.resolve(routeId, authorization, filter, wrapRoute);
+        final MessagePredicate emptyFilter = (t, b, o, l) -> true;
```

Comment:
This is not really an _empty_ filter, as one could say an empty filter causes the result to be empty.
Suggest renaming to `matchAny` and also hoisting to a constant.